### PR TITLE
Feature/calendar screen agenda

### PR DESCRIPTION
**Pull request by serjKim**

### CODE REVIEW

File: mobile/src/calendar/events-editor-button.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { TouchableOpacity, ViewStyle, LayoutChangeEvent, StyleSheet, View } from 'react-native';
+import { StyledText } from '../override/styled-text';
+import { eventsEditorStyles } from './styles';
+
+export type OnPressEventsEditorButton = (isRequest: boolean) => void;
+
+interface EventsEditorButtonProps {
+    requestTitle: string;
+    reviewTitle: string;
+    onPress: OnPressEventsEditorButton;
```
Comment (santee) [inline mobile/src/calendar/events-editor-button.tsx]: onClick :)

File: mobile/src/calendar/calendar-actions-button-group.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    selectedCalendarDay: DayModel;
+}
+
+export class ActionsButtonGroupImpl extends Component<ActionButtonGroupProps> {
+    public render() {
+        const { vacation, dayoff, sickleave } = this.getIntervals();
+
+        return (
+            <View style={calendarActionsStyles.container}>
+                <VacationActionButton interval={vacation} />
```
Comment (demkalkov) [inline mobile/src/calendar/calendar-actions-button-group.tsx]: If you are trying to vertically align the buttons, then you should add one more `CalendarActionButtonSeparator` before vacation button

File: mobile/src/calendar/calendar-actions-button-group.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        let vacation: IntervalModel;
+        let dayoff: IntervalModel;
+        let sickleave: IntervalModel;
+
+        if (intervalsModel) {
+             const intervals = intervalsModel.get(selectedCalendarDay.date);
+
+             if (intervals) {
+                vacation = intervals.find(x => x.eventType === CalendarEventsType.Vacation);
+                dayoff = intervals.find(x => x.eventType === CalendarEventsType.Dayoff);
```
Comment (santee) [inline mobile/src/calendar/calendar-actions-button-group.tsx]: dayoff *or* workout. That name can also be generalized in code as 'workHoursChange'

File: mobile/src/calendar/calendar-actions-button-group.tsx (hunk 2 above)
Comment (serjKim) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: I don't need separator before vacation button, there is now align in the sketch

File: mobile/src/calendar/calendar-actions-button-group.tsx (hunk 3 above)
Comment (serjKim) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: let's leave it. We use dayoff everywhere

**Pull request merged**